### PR TITLE
Types with no doc block

### PR DESCRIPTION
**Pull request by marcosh**
solves #229

### CODE REVIEW

File: test/unit/TypesFinder/FindParameterTypeTest.php [hunk 1]
```diff
@@ -127,4 +127,18 @@ public function testFindParameterTypeForMethod($docBlock, $nodeName, $expectedIn
...
     }
+
+    public function testFindParameterTypeForFunctionWithNoDocBlock()
+    {
+        $node = new ParamNode('foo');
+
+        $function = $this->createMock(ReflectionFunction::class);
+
+        $function
+            ->expects($this->once())
```
Comment (asgrim) [inline test/unit/TypesFinder/FindParameterTypeTest.php]: `self::once()`

File: test/unit/TypesFinder/FindParameterTypeTest.php [hunk 2]
```diff
@@ -127,4 +127,18 @@ public function testFindParameterTypeForMethod($docBlock, $nodeName, $expectedIn
...
+            ->method('getDocComment')
+            ->will($this->returnValue(''));
```
Comment (asgrim) [inline test/unit/TypesFinder/FindParameterTypeTest.php]: `self::returnValue`

**Comment by marcosh** [comment]
@asgrim thanks for your review. I fixed what you suggested. Sorry for the non static methods, I just looked how they where used elsewhere in the project...

**Comment by asgrim** [comment]
@marcosh yep we need to change them :)

**Comment by asgrim** [comment]
Thanks @marcosh! :)

**Pull request merged**